_content/av/visuddhimagga-for-sutta-lovers-part-2_sujato.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "The Visuddhimagga for Sutta Lovers"
+authors:
+  - sujato
+source_url: "https://learn.dhammanet.org/courses/introductory/the-visuddhimagga-for-sutta-lovers/"
+drive_links:
+  - "https://drive.google.com/file/d/1yjGOk_xOUwqRBMbQBvxdlhs6YxSpboxw/view?usp=drive_link"
+formats: 
+- "Part 2 mp4"
+course: theravada-roots
+tags:
+  - vsm
+  - path
+year: 2020
+month: may
+series: visudhimagga-for-sutta-lovers
+---
+
+This four-part series on the Visuddhimagga, led by Bhante Sujato in May 2020, covers the fundamental aspects of Buddhist practice and theory in 5th century Theravāda Buddhism.
```

Comment:
A fine description for the series. Each entry gets a unique description describing it specifically.